app/src/main/java/ai/brokk/gui/WorkspaceChip.java

```diff
@@ -1047,45 +979,35 @@ private String buildAggregateSummaryTooltip() {
                     .sorted()
                     .toList();
 
-            StringBuilder body = new StringBuilder();
-
             int totalLoc = 0;
             int totalTokens = 0;
-            try {
-                for (var summary : summaryFragments) {
-                    String text;
-                    if (summary instanceof ContextFragment.ComputedFragment cf) {
-                        text = cf.computedText().renderNowOr("");
-                    } else {
-                        text = summary.text();
-                    }
-                    totalLoc += text.split("\\r?\\n", -1).length;
-                    totalTokens += Messages.getApproximateTokens(text);
-                }
-                body.append("<div>")
-                        .append(formatCount(totalLoc))
-                        .append(" LOC \u2022 ~")
-                        .append(formatCount(totalTokens))
-                        .append(" tokens</div><br/>");
-            } catch (Exception e) {
-                logger.error(e);
-            }
-
-            body.append("<div><b>Summaries</b></div>");
-            body.append("<hr style='border:0;border-top:1px solid #ccc;margin:4px 0 6px 0;'/>");
-
-            if (allFiles.isEmpty()) {
-                body.append("Multiple summaries");
-            } else {
-                body.append("<ul style='margin:0;padding-left:16px'>");
-                for (var f : allFiles) {
-                    body.append("<li>").append(StringEscapeUtils.escapeHtml4(f)).append("</li>");
+            for (var summary : summaryFragments) {
+                String text;
+                if (summary instanceof ContextFragment.ComputedFragment cf) {
+                    text = cf.computedText().renderNowOr("");
```

Comment:
Similarly here, do we have a plan for when we hit the "blank string" outcome? I know this was from previous code, so maybe this is a follow-up PR.